test/lakefsfs/build.sbt

```diff
@@ -26,11 +29,10 @@ def generateProject(buildType: BuildType) =
       settingsToCompileIn(),
       scalaVersion := buildType.scalaVersion,
       libraryDependencies ++= Seq(
-        "io.lakefs" % "api-client" % "0.40.3",
+        "io.lakefs" % "hadoop-lakefs-assembly" % "0.1.0-RC.0-SNAPSHOT",
```

Comment:
Can you document in the PR (say in a response here) why this works?  Just in case we run into this again ;-(

Reply:
Updated the build.sbt with a comment on why we used the lakefs-assembly